.php_cs.dist

```diff
@@ -46,13 +100,13 @@ return \PhpCsFixer\Config::create()
             'commentType' => 'PHPDoc',
             'header' => $header,
             'location' => 'after_open',
-            'separate' => 'bottom',
+            'separate' => 'both',
```

Comment:
I wonder if this extra line above the header comment is really necessary.

Reply:
It's not necessary per se but I think it's nicer & more consistent: for pretty much every block we have a blank line on both sides:

```php
<php

/**
 * ...
 */

declare(strict_types=1);

namespace Acme;
```

Instead of:

```php
<php
/**
 * ...
 */

declare(strict_types=1);

namespace Acme;
```

Although tbh my personal preference would be:

```php
<php declare(strict_types=1);

/*
 * ...
 */

namespace Acme;
```

Reply:
The thing is that without this extra line header is very tidy:

![Screenshot from 2019-06-27 16-51-58](https://user-images.githubusercontent.com/139488/60247591-e522ff00-98fb-11e9-8c03-2bdd759baf73.png)

> `<php declare(strict_types=1);`
This totally breaks header comment folding in Eclipse.

Reply:
I find it as tidy tbh :) Also worth mentioning it's actually PHP-CS-Fixer default. But anyway since it's such a source of debate let's revert that.

>This totally breaks header comment folding in Eclipse.

 I'm not surprised: I found Eclipse to be really lacking and lagging behind when it comes to PHP support (worth mentioning this declaration is valid even in 5.x)... Which is why I switched over to PhpStorm which is still at a very reasonable price for the time I spent on it. It is nonetheless a bug you may want to report